### PR TITLE
Renaming Service APIs to Gateway API

### PR DESCRIPTION
**Pull request by robscott**
This accompanies https://github.com/kubernetes-sigs/service-apis/pull/541 and related discussion in https://github.com/kubernetes-sigs/service-apis/issues/536. Not quite sure what the correct order of operations here is.

### CODE REVIEW

File: communication/slack-config/sig-network/config.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 channels:
-  - name: sig-network-service-apis
+  - name: sig-network-gateway-api
```
Comment (mrbobbytables) [inline communication/slack-config/sig-network/config.yaml:2]:
```suggestion
  - name: sig-network-gateway-api
    id: CR0H13KGA
```

When renaming a channel, the underlying channel ID is needed 👍

**Review by mrbobbytables — CHANGES_REQUESTED** [review]

File: communication/slack-config/sig-network/config.yaml (hunk 1 above)
Comment (robscott) [reply communication/slack-config/sig-network/config.yaml:2]: Updated, thanks!

File: mentoring/programs/office-hours.md [hunk 2]
```diff
@@ -14,7 +14,5 @@ title: Office Hours/ Mentoring
     * [kops Office Hours - Biweekly Friday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit)
     * [kubeadm Office Hours - Biweekly Wednesday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/130_kiXjG7graFNSnIAgtMS1G8zPDwpkshgfRYS0nggo/edit)
     * [minikube Office Hours - Biweekly Monday at 11:00 PT (Pacific Time)](https://docs.google.com/document/d/1jhfmL1gsgN39uCEgz5pW9tnIotFgHhxq2yfMK3KYE4w/edit)
-* Network -
-    * [Service APIs Office Hours - Biweekly Wednesday at 15:00 PT (Pacific Time)](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit)
```
Comment (robscott) [inline mentoring/programs/office-hours.md:18]: This was an intentional removal - we don't have office hours anymore. Looks like I missed this when we moved our meetings around.

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**